### ISSUE TITLE
Pressing all the way up on a controller stick results in the value flipping to negative

### ISSUE THREAD

**Issue by prime31** [mono/MonoGame#4711]
To reproduce:

- add the following line in Update:
`Console.WriteLine( "pressing up: {0}", GamePad.GetState( 0 ).ThumbSticks.Left );`
- slowly press up on the left thumb stick and watch the printed values

Here is example output:

```
Log: pressing up: {X:0 Y:0.5549116}
Log: pressing up: {X:0 Y:0.5898071}
Log: pressing up: {X:0 Y:0.6072749}
Log: pressing up: {X:0 Y:0.6072749}
Log: pressing up: {X:0 Y:0.6072749}
Log: pressing up: {X:0 Y:0.6203657}
Log: pressing up: {X:0 Y:0.6203657}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6247428}
Log: pressing up: {X:0 Y:0.6378336}
Log: pressing up: {X:0 Y:0.7032878}
Log: pressing up: {X:0 Y:0.7469373}
Log: pressing up: {X:0 Y:0.764365}
Log: pressing up: {X:0 Y:0.8429501}
Log: pressing up: {X:0 Y:0.8429501}
Log: pressing up: {X:0 Y:0.856041}
Log: pressing up: {X:0 Y:0.8996904}
Log: pressing up: {X:0 Y:0.9389629}
Log: pressing up: {X:0 Y:0.9520538}
Log: pressing up: {X:0 Y:0.9651446}
Log: pressing up: {X:0 Y:-1}
Log: pressing up: {X:0 Y:-1}
```

I have only tested this on Mac.

**Comment by tomspilman** [comment]
@cra0zy @dellis1972 - This sounds like an OpenTK issue of some sort?

**Comment by cra0zy** [comment]
Already fixed in 3.5.1

**Comment by tomspilman** [comment]
Ah....  @prime31 can you download the 3.5.1 release and retest?

**Comment by prime31** [comment]
Fixed in 3.5.1! You guys are fast as hell! I didnt see that on the change log so I didnt even think to check. Glad I posted before digging into that hot mess that OpenTK is ;)
